src/Mono.Android/methodmap.csv

```diff
@@ -78,7 +78,6 @@
 5, android.bluetooth,BluetoothClass,getMajorDeviceClass,return,Android.Bluetooth.MajorDeviceClass
 5, android.bluetooth,BluetoothClass,hasService,service,Android.Bluetooth.ServiceClass
 5, android.bluetooth,BluetoothDevice,getBondState,return,Android.Bluetooth.Bond
-0, android.content,Context,bindService,flags,Android.Content.Bind
```

Comment:
I assume that these removals were made because they were "duplicative" of the `metadata` additions, e.g.

```xml
<attr 
    path="/api/package[@name='android.content']/class[@name='Context']/method[@name='bindService' and count(parameter)=3 and parameter[1][@type='android.content.Intent'] and parameter[2][@type='android.content.ServiceConnection'] and parameter[3][@type='int']]/parameter[@name='flags']" 
    name="enumType">Android.Content.Bind</attr>
```

Is my assumption correct?

Is there a reason to prefer the `metadata` approach vs. the `methodmap.csv` approach?

Reply:
My bad, the `metadata` lines were placed in a section that describes why they are used, but that doesn't show up in the PR diff: https://github.com/xamarin/xamarin-android/pull/7980/files#diff-dbf22a7169e51cdd444864d89c196068bd583e320486401e528bf3b28917569cL1670-L1672

```
  <!-- We have 2 methods with the same name and the same parameter name. One needs to be enumified but not the other.
       Our methodmap.csv isn't capable of only choosing 1 of the methods, so it sets them both as the enum, making both have the
       same signature. So we are going to set the enum transformation here instead, where we can specify a path that only affects a single method. -->
```